modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/hooks.ts

```diff
@@ -0,0 +1,85 @@
+import {useContext, useLayoutEffect} from 'react';
+import {useUniqueId} from '@workday/canvas-kit-react-common';
+import {usePopup, useCloseOnOutsideClick, useCloseOnEscape} from '@workday/canvas-kit-react-popup';
+
+import {DropdownContext} from './Provider';
+import {DropdownMenuProps} from './Menu';
+import {DropdownButtonProps} from './Button';
+
+export const useFocusActiveItemElement = <E extends HTMLElement>(
+  activeDropdownItemRef: React.RefObject<E>
+) => {
+  useLayoutEffect(() => {
+    if (activeDropdownItemRef.current) {
+      return activeDropdownItemRef.current.focus();
+    }
+  });
+};
+
+export const useDropdown = (
+  activeDropdownItemRef: React.RefObject<HTMLAnchorElement>,
+  buttonRef: React.RefObject<HTMLButtonElement>,
+  buttonId: string = ''
```

Comment:
Nit: I don't think this default parameter is needed here. `undefined` should get you a generated id.